src/js/providers/html5.js

```diff
@@ -129,15 +128,6 @@ define([
 
             _fullscreenState = false;
 
-        // Overwrite the event dispatchers to block on certain occasions
-        this.sendEvent = function() {
-            if (!_attached) {
```

Comment:
Again, this functionality has been removed. Events should not be triggered if _attached is false.